### DISCUSSION TITLE
Orario docenti

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#148] [Idee e proposte]
Una proposta che secondo me sarebbe utile per il registro elettronico giuaschool sarebbe quella di inserire su piattaforma un orario da cui poi cliccando sopra l'ora in cui bisogna fare lezione si accede direttamente al registro firme della classe

**Comment by sempliciniluigi89** [comment]
Concordo,come usato da Argo e Classeviva

**Comment by trinko** [comment]
La segno come richiesta.
